react/features/lobby/actions.web.js

```diff
@@ -0,0 +1,209 @@
+// @flow
+
+import { type Dispatch } from 'redux';
+
+import { appNavigate, maybeRedirectToWelcomePage } from '../app';
+import { conferenceLeft, conferenceWillJoin, getCurrentConference, setPassword } from '../base/conference';
+import { hideDialog, openDialog } from '../base/dialog';
+import { getLocalParticipant } from '../base/participants';
+
+import {
+    KNOCKING_PARTICIPANT_ARRIVED_OR_UPDATED,
+    KNOCKING_PARTICIPANT_LEFT,
+    SET_KNOCKING_STATE,
+    SET_LOBBY_MODE_ENABLED,
+    SET_PASSWORD_JOIN_FAILED
+} from './actionTypes';
+import { LobbyScreen } from './components';
+
+declare var APP: Object;
+
+/**
+ * Cancels the ongoing knocking and abandones the join flow.
+ *
+ * @returns {Function}
+ */
+export function cancelKnocking() {
+    return async (dispatch: Dispatch<any>, getState: Function) => {
+        if (typeof APP !== 'undefined') {
+            // when we are redirecting the library should handle any
+            // unload and clean of the connection.
+            APP.API.notifyReadyToClose();
+            dispatch(maybeRedirectToWelcomePage());
+
+            return;
+        }
+
+        dispatch(conferenceLeft(getCurrentConference(getState)));
```

Comment:
No, it's not

Reply:
Yes it is: https://github.com/jitsi/jitsi-meet/blob/338c96021568d08eaac9e26d6224a7b7976487e8/react/features/app/actions.js#L79

If a conference exists it will leave it and if it doesn't it will disconnect XMPP. Both of those would result in a CONFERENCE_TERMINATED event on the SDK side.